.travis.yml

```diff
@@ -16,23 +16,17 @@ dist: trusty
 addons:
   apt:
     packages:
+    # for bashate
+    - python-tz
     # for checkbashisms
     - devscripts
 
 # Note: if statements cannot go across multiple lines. Combine into one, or move to a shell script!
 
-# On Ubuntu, shellcheck is not in trusty, but is in trusty-backports, but those aren't available in TravisCI..)
-# Using artful to get 0.4.6, which has support for disabling a check globally within a file
+# On Ubuntu, shellcheck is already available in TravisCI
 before_install:
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then echo "deb http://archive.ubuntu.com/ubuntu/ artful universe" | sudo tee -a /etc/apt/sources.list ; fi
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then time sudo apt-get -q update ; fi
-    - if [[ "$TRAVIS_OS_NAME" == "linux" ]] ; then time sudo apt-get install python-bashate shellcheck ; fi
-
```

Comment:
Please add a comment about why you're installing via this script.

Reply:
Done.